docs/userGuide/deployingTheSite.md

```diff
@@ -49,6 +49,40 @@ You can override the default deployment settings %%(e.g., repo/branch to deploy)
 `markbind deploy` does not generate the static site from your source; it simply deploys the files that are already in the `_site` directory. You need to run `markbind build` first if you want to generate the site before deploying.
 </box>
 
+### Deploying to GitHub Pages via Travis CI
+**You can setup [<tooltip content="a platform for Continous Integration and Delivery (among other things)">Travis CI</tooltip>](https://www.travis-ci.org/) to automatically build and deploy your site on GitHub Pages every time your GitHub repo is updated.**
+
+Here are the steps to set up Travis CI:
+
+1. [Sign up with GitHub](https://travis-ci.com/signin) at [Travis CI](https://travis-ci.org).
+1. Accept the autorisation for Travis CI when you are redirected to GitHub.
+1. Click the green _Activate_ button, and select the repository with the MarkBind site.
+1. [Generate a GitHub personal access token](https://help.github.com/articles/creating-a-personal-access-token-for-the-command-line/#creating-a-token) with **repo** permissions. Take note of the generated token - you will not be able to see it again once you navigate away from the page.
+1. [Add an environment variable in Travis CI](https://docs.travis-ci.com/user/environment-variables/#defining-variables-in-repository-settings) named `GITHUB_TOKEN`, with the value set to the personal access token generated in the previous step. Ensure that _Display value in the build log_ is set to _Off_.
+1. Add a `.travis.yml` file to instruct Travis CI to run `markbind deploy --travis` when you push to the repository. An example `.travis.yml` file that can accomplish this is given below:
+    <box background-color="white" border-color="white">
+    ```yaml
+    language: node_js
+    node_js:
+      - '8'
+    install:
+      - npm i -g markbind-cli
+    script: markbind deploy --travis
```

Comment:
Is it possible to do a markbind build && markbind deploy --travis here? That way deploy would not have to generate. The `-t` option would simply mean deploying in a travis environment?